### DISCUSSION TITLE
Mod selection to be compatible with stable

### DISCUSSION THREAD

**Discussion by Minorsonek** [ppy/osu#21704] [Ideas] [answered]
Hi, so in the osu stable, when it comes to mod selection shortcuts, we are used to hit "F" to select hidden.
In lazer, it selects double time, which is pretty annoying for lifetime osu gamers and it is really hard to adjust to new way of selecting mods after years of selecting +HD as "F".

I would like to propose two solutions, either let us remap the hotkeys, or just be compatible with osu stable, so for example, don't use "G" for nightcore, but just "D" two times as in stable, same for +SD and +PF, so it will bring +HD back at "F".

**Comment by Theighlin** [comment, marked as answer]
There's a setting to enable legacy keybinds for the mod select menu in the options.

**Reply by Minorsonek** [reply]
Thanks, love this!
